### ISSUE TITLE
Add `icon`, `ghost_head`, and `ghost_foot` to `data-stub.json` file

### ISSUE THREAD

**Issue by AileenCGN** [TryGhost/gatsby-starter-ghost#18]
Because the publication icon and the code injections are not populated by default for new Ghost instances, they are populated as `null` from the [gatsby-source-ghost](https://github.com/TryGhost/gatsby-source-ghost/) plugin. Using these properties then in GraphQL queries causes the build to break.

This is a known issue with GraphQL and being tracked for this repository here: https://github.com/TryGhost/gatsby-source-ghost/issues/2

Due to this issue we created a [data-stub](https://gist.github.com/AileenCGN/172ed94bcd18a328034e0259dbf3e702) file which needs to be imported before you connect the starter with your own Ghost installation.

This data-stub file needs to be amended to insert an icon (preferably `favicon.png`), as well as some dummy site codeinjection content.

Whereas this change is pretty easy to do, it needs to be discussed and thought through, what data we want to inject and how to prevent overwriting existing one.

Once, the data-stub file is extended, we need to include those three properties in the [`fragments`](https://github.com/TryGhost/gatsby-starter-ghost/blob/master/src/utils/fragments.js#L216) file.

**Comment by AileenCGN** [comment]
The main issue I see here is that the chance of someone having a custom icon || codeinjection already in place is very high. Adding those values to the data-stub file would cause it to overwrite their custom data which is not a nice solution.

One - very simple - solution could just be to write instructions in our README and explain that those fields need to be populated manually by the user.

Even tho `icon` is _currently_ not used in this starter by default, it should be available at least. 

Depending on the outcome of #20 the usage of `ghost_head` and `ghost_foot` might not be needed within our GraphQL schema, as it will be available in the global site config.

**Comment by ErisDS** [comment]
I think logo is also affected?

**Comment by AileenCGN** [comment]
> I think logo is also affected?

Yes, can be. I didn't include this initially, as it gets populated by default for new Ghost installs.
